core/templates/components/ck-editor-helpers/ck-editor-copy-toolbar/ck-editor-copy-toolbar.component.ts

```diff
@@ -53,6 +53,10 @@ export class CkEditorCopyToolbarComponent {
         });
     }
   }
+
+  ngOnDestroy(): void {
+    this.document.body.style.cursor = '';
```

Comment:
Can we ad a unit test for this function?

Reply:
Sure.

Reply:
I have written the test for this! Do have a look.